src/client/HTTP_request.ts

```diff
@@ -23,14 +24,14 @@ export class HTTP_request {
                            atf_filename: string, atf_text: string) {
         /* Send attachment to server containing ATF file and necessary data to
         run given command (validate, lemmatise, etc).*/
-        let responseID: string;
+        //let responseID: string;
```

Comment:
This file isn't being used at all at the moment and I was only updating it enough so that it compiles... I suspect the whole thing will be deleted, or otherwise restructured.